src/lib.rs

```diff
@@ -605,18 +624,20 @@ pub struct Arrow {
 
 use self::ArrowShape::*;
 
-impl Arrow {
-    /// Return `true` if this is a default arrow.
-    fn is_default(&self) -> bool {
-        self.arrows.is_empty()
-    }
-
+impl Default for Arrow {
     /// Arrow constructor which returns a default arrow
-    pub fn default() -> Arrow {
```

Comment:
Clippy now lints against `fn default() -> Self` not in a `Default` implementation. While the crate is pre-1.0 and this is very unlikely to affect anyone, this is technically a breaking change, so I can revert if you'd prefer